lib/src/components/day_view_components.dart

```diff
@@ -157,7 +164,7 @@ class DefaultTimeLineMark extends StatelessWidget {
       child: Padding(
         padding: const EdgeInsets.only(right: 7.0),
         child: Text(
-          "${((date.hour - 1) % 12) + 1} ${date.hour ~/ 12 == 0 ? "am" : "pm"}",
+          timeStringBuilder(date),
```

Comment:
Check for null condition here. and use previous format if timeStringBuilder is null.